packages/plugins/plugin-sheet/src/SheetPlugin.tsx

```diff
@@ -144,22 +144,19 @@ export const SheetPlugin = (): PluginDefinition<SheetPluginProvides> => {
       surface: {
         component: ({ data, role }) => {
           const space = isEchoObject(data.object) ? getSpace(data.object) : undefined;
-          const graph = useComputeGraph(space);
-          const handleSelectRange = () => {};
-          const handleDeleteRange = () => {};
 
           switch (role) {
             case 'article':
             case 'section':
-              if (graph && data.object instanceof SheetType) {
-                return <SheetContainer graph={graph} sheet={data.object} role={role} />;
+              if (space && data.object instanceof SheetType) {
```

Comment:
maybe return early if no space (since other things could technically work, but we're def in an invalid state)?